### PR TITLE
New: Read-only state for TreePicker

### PR DESCRIPTION
**Pull request by patrick-ivers**
#### Changes
- Set TreePickerNode buttons to disabled when TreePicker is disabled
- Make Breadcrumb disappear when TreePicker is disabled

#### Trello
https://trello.com/c/ciNHr1ys/5426-adslot-ui-add-support-for-read-only-listpicker

#### Screenshots
screen shot 2016-06-24 at 2 26 53 pm

### CODE REVIEW

**Comment by vinteo** [comment]
LGTM

File: src/components/adslotUi/TreePickerNodeComponent.js [hunk 1]
```diff
@@ -24,13 +24,13 @@ const getExpander = ({ expandNode, node }) => {
 };
 
 const TreePickerNodeComponent = ({
-  expandNode,
+  disabled,
   includeNode,
+  expandNode,
```
Comment (OndrejCholeva) [inline src/components/adslotUi/TreePickerNodeComponent.js]: e < i

**Comment by sharmajyo** [comment]
LGTM

File: test/components/adslotUi/TreePickerNodeComponentTest.js [hunk 2]
```diff
@@ -74,6 +75,17 @@ describe('TreePickerNodeComponent', () => {
     expect(buttonElement.children().text()).to.equal('−');
   });
 
+  it('should render button as disabled when disabled is true', () => {
+    let fireCount = 0;
+    const testFunction = () => { fireCount += 1; };
```
Comment (OndrejCholeva) [inline test/components/adslotUi/TreePickerNodeComponentTest.js]: I think we mostly use `fireCount++;`, but I am fine if you leave it.

Comment (patrick-ivers) [inline test/components/adslotUi/TreePickerNodeComponentTest.js]: Ah yea.. I copy/pasted that from another test. I'll keep it in for the time being for consistency.

**Pull request merged**